storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11208,6 +11209,85 @@ static void show_myrocks_vars(THD *thd, SHOW_VAR *var, char *buff) {
   var->value = reinterpret_cast<char *>(&myrocks_status_variables);
 }
 
+static ulonglong
+io_stall_prop_value(const std::map<std::string, std::string> &props,
+                    std::string &&key) {
+  std::map<std::string, std::string>::const_iterator iter =
+      props.find("io_stalls." + key);
+  if (iter != props.end()) {
+    return std::stoull(iter->second);
+  } else {
+    return 0;
+  }
+}
+
+static void update_rocksdb_stall_status() {
+  io_stall_stats = {};
```

Comment:
Initializing io_stall_stats and then adding counters to it might cause strange behavior if the command is run concurrently. It might be better to not initialize io_stall_stats and set the members equal to the value returned by rocksdb, similar to the other status variables in memory_usage or export stats.

Reply:
It is a bit different than memory_usage or export stats methods as I need to loop through multiple calls to accumulate the final result. But I see you concern, so I can create a local struct then copy it to the global one at the end of method call.